x-pack/plugins/security_solution/public/detections/pages/alert_details/tabs/summary/user_panel/index.tsx

```diff
@@ -0,0 +1,160 @@
+import {
+  IP_ADDRESSES_TITLE,
+  LAST_SEEN_TITLE,
+  USER_NAME_TITLE,
+  USER_PANEL_TITLE,
+  USER_RISK_CLASSIFICATION,
+  USER_RISK_SCORE,
+} from '../translation';
+import { SummaryPanel } from '../wrappers';
+import { UserPanelActions, USER_PANEL_ACTIONS_CLASS } from './user_panel_actions';
+
+export interface UserPanelProps {
+  data: TimelineEventsDetailsItem[] | null;
+  selectedPatterns: SelectedDataView['selectedPatterns'];
+  openUserDetailsPanel: (userName: string, onClose?: (() => void) | undefined) => void;
+}
+
+const UserPanelSection: React.FC<{
+  title?: string | React.ReactElement;
+  grow?: FlexItemGrowSize;
+}> = ({ grow, title, children }) =>
+  children ? (
+    <EuiFlexItem grow={grow}>
+      {title && (
+        <>
+          <EuiTitle size="xxs">
+            <h5>{title}</h5>
+          </EuiTitle>
+          <EuiSpacer size="xs" />
+        </>
+      )}
+      {children}
+    </EuiFlexItem>
+  ) : null;
+
+export const UserPanel = React.memo(
+  ({ data, selectedPatterns, openUserDetailsPanel }: UserPanelProps) => {
+    const userName = useMemo(() => getTimelineEventData('user.name', data), [data]);
+
+    const { data: userRisk, isLicenseValid: isRiskLicenseValid } = useRiskScore({
+      riskEntity: RiskScoreEntity.user,
+      skip: userName == null,
+    });
+
+    const renderUserActions = useCallback(
+      () => <UserPanelActions openUserDetailsPanel={openUserDetailsPanel} userName={userName} />,
+      [openUserDetailsPanel, userName]
+    );
+
+    const [userRiskScore, userRiskLevel] = useMemo(() => {
+      const userRiskData = userRisk && userRisk.length > 0 ? userRisk[0] : undefined;
+      const userRiskValue = userRiskData
+        ? Math.round(userRiskData.user.risk.calculated_score_norm)
+        : getEmptyTagValue();
+      const userRiskSeverity = userRiskData ? (
+        <RiskScore severity={userRiskData.user.risk.calculated_level} hideBackgroundColor />
+      ) : (
+        getEmptyTagValue()
+      );
+
+      return [userRiskValue, userRiskSeverity];
+    }, [userRisk]);
+
+    const sourceIpFields = useMemo(
+      () => find({ field: 'source.ip', category: 'source' }, data)?.values ?? [],
+      [data]
+    );
+
+    const renderSourceIp = useCallback(
+      (ip: string) => (ip != null ? <NetworkDetailsLink ip={ip} /> : getEmptyTagValue()),
+      []
+    );
+
+    return (
+      <SummaryPanel
+        actionsClassName={USER_PANEL_ACTIONS_CLASS}
+        grow
+        renderActionsPopover={userName ? renderUserActions : undefined}
+        title={USER_PANEL_TITLE}
+      >
+        <EuiFlexGroup data-test-subj="user-panel">
+          <EuiFlexItem grow={2}>
+            <EuiFlexGroup>
+              <UserPanelSection grow={false}>
+                <EuiIcon type="userAvatar" size="xl" />
+              </UserPanelSection>
+              <UserPanelSection title={USER_NAME_TITLE}>
+                <UserDetailsLink userName={userName} />
```

Comment:
Seems that user.name is not always available, we might want a place holder if it is not available.
<img width="1992" alt="Screenshot 2022-10-28 at 18 37 33" src="https://user-images.githubusercontent.com/6295984/198698683-914e4fd3-865f-4556-98a1-325a8b5987d0.png">

Reply:
fixed, thanks!